hedera-node/src/main/java/com/hedera/services/ServicesState.java

```diff
@@ -174,9 +178,12 @@ public int getVersion() {
 
 	@Override
 	public int getMinimumChildCount(int version) {
-		if (version >= MINIMUM_SUPPORTED_VERSION && version <= CURRENT_VERSION) {
+		if (version >= MINIMUM_SUPPORTED_VERSION && version < CURRENT_VERSION) {
 			return NUM_POST_0210_CHILDREN;
-		} else {
+		} else if (version == CURRENT_VERSION) {
+			return NUM_POST_0270_CHILDREN;
```

Comment:
This should be renamed to `NUM_0270_CHILDREN`

Reply:
Fixed, also bumped minimum supported version to 0.25.x